backend/src/Database.hs

```diff
@@ -63,36 +70,65 @@ initializeDatabase = do
           \country TEXT,\
           \PRIMARY KEY(id)\
           \);"
+        execute_
+          conn
+          "CREATE TABLE Ratings (\
+          \id INTEGER NOT NULL,\
+          \playerId INTEGER NOT NULL,\
+          \side TEXT NOT NULL,\
+          \timestamp INTEGER NOT NULL,\
+          \reportId INTEGER NOT NULL,\
+          \ratingBefore INTEGER NOT NULL,\
+          \ratingAfter INTEGER NOT NULL,\
```

Comment:
You were debating whether to include one or both before, did this just end up being the simpler choice to start with, and can get fancy later?

Reply:
I suspect/realized `ratingBefore` (which is currently unnecessary) will make historical ratings adjustments much easier to do if/when those become necessary

Reply:
Because there will be no need to "find the last game played prior to the one you're adjusting"

Reply:
Good point 💯